app/views/transient_registrations/show.html.erb

```diff
@@ -138,10 +133,32 @@
             </p>
           </div>
         </div>
-
-
-
-
+        <hr/>
+        <h2 class="heading-medium"><%= t(".conviction_heading") %></h2>
+        <div>
+          <% if @transient_registration.conviction_check_approved? %>
+            <%= link_to t(".conviction_link"), transient_registration_convictions_path(@transient_registration.reg_identifier), class: "pull-up"  %>
+            <p>
+              <%= t(".conviction_search_text.approved") %>
+            </p>
+          <% elsif @transient_registration.metaData.REVOKED? %>
+            <%= link_to t(".conviction_link"), transient_registration_convictions_path(@transient_registration.reg_identifier), class: "pull-up"  %>
+            <p>
+              <%= t(".conviction_search_text.rejected") %>
+            </p>
+          <% elsif @transient_registration.conviction_search_result.present? %>
+            <% if @transient_registration.conviction_check_required? %>
+              <%= link_to t(".conviction_link"), transient_registration_convictions_path(@transient_registration.reg_identifier), class: "pull-up" %>
+            <% end %>
+            <p>
+              <%= t(".conviction_search_text.#{@transient_registration.conviction_check_required?}") %>
+            </p>
+          <% else %>
+            <p>
+              <%= t(".conviction_search_text.unknown") %>
+            </p>
+          <% end %>
```

Comment:
I could be wrong, but I feel this is a good candidate for moving into a helper. From what I can see, we're currently checking various properties of the `transient_registration` to determine 2 things

- should we show a link
- what text to show `t(".conviction_search_text.[KEY]")`

I think now is a good opportunity to move these into helper methods (which we can then unit test) and simplify the view.

```ruby
<% if display_convictions_link?(@transient_registration) %>
  <%= link_to t(".conviction_link"), transient_registration_convictions_path(@transient_registration.reg_identifier), class: "pull-up"  %>
<% end %>
<p>
  <%= convictions_text(@transient_registration) %>
</p>
```

Reply:
That's fine, and with something, as complex as the view details page, we probably need a presenter more than a bunch of helpers.

So agree to bite my tongue and we'll aim to return to this view later.